public/mix-manifest 2.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
please remove this.
This is not doing anything

Reply:
I removed it.